docs/guides/vault/create-a-vault.md

```diff
@@ -11,29 +14,205 @@ This guide will teach you how to deploy an **[LSP9Vault](../../standards/smart-c
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```shell title="Install the dependencies"
+npm install web3 @lukso/lsp-smart-contracts
+```
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```shell title="Install the dependencies"
+npm install ethers @lukso/lsp-smart-contracts
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Step 1 - Setup imports and constants​
+
+For the imports, we will need the `LSP9vault` contract **ABI** and **bytecode** in order to deploy a LSP9 Vault.  
+For the contants we will need the _private key_ and _the address of the vault receiver_.
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```typescript title="Imports & Constants"
+import LSP9Vault from '@lukso/lsp-smart-contracts/artifacts/LSP9Vault.json';
 import Web3 from 'web3';
+
+const web3 = new Web3('https://rpc.l16.lukso.network');
+const vaultReceiver = '0x...'; // The address that will be the vault owner
+
+// initialize your EOA
+const privateKey = '0x...';
+const myEOA = web3.eth.accounts.wallet.add(privateKey);
+```
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```typescript title="Imports & Constants"
+import LSP9Vault from '@lukso/lsp-smart-contracts/artifacts/LSP9Vault.json';
+import { ethers } from 'ethers';
+
+const provider = new ethers.JsonRpcProvider('https://rpc.l16.lukso.network');
+const vaultReceiver = '0x...'; // The address that will be the vault owner
+
+// setup your EOA
+const privateKey = '0x...';
+const myEOA = new ethers.Wallet(privateKey).connect(provider);
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Step 2 - Instantiate contracts
+
+Create instance for `LSP9Vault`, that is needed in order to deploy the contract.
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```typescript title="Contract instance"
+// create an instance of the LSP9Vault contract
+let myVault = new web3.eth.Contract(LSP9Vault.abi);
+```
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```typescript title="Contract instance"
+// create an factory for the LSP9Vault contract
+let vaultFactory = new ethers.ContractFactory(
+  LSP9Vault.abi,
+  LSP9Vault.bytecode,
+);
+```
+
+  </TabItem>
+
+</Tabs>
+
+## Step 3 - Send transaction
+
+Finally send the **contract deployment** transaction and in a few seconds the contract may be used.
+
+<Tabs>
+  
+  <TabItem value="web3js" label="web3.js">
+
+```typescript title="Sending contract deployment transaction"
+// deploy the vault contract
+await myVault
+  .deploy({
+    data: LSP9Vault.bytecode,
+    arguments: [vaultReceiver],
+  })
+  .send({
+    from: myEOA.address,
+    gas: 5_000_000,
+    gasPrice: '1000000000',
+  });
+```
+
+  </TabItem>
+
+  <TabItem value="ethersjs" label="ethers.js">
+
+```typescript title="Sending contract deployment transaction"
+// deploy the vault contract
+const myVault = await vaultFactory.connect(myEOA).deploy(vaultReceiver);
```

Comment:
Okay I see its the owner. In that case lets call it the owner and not use the word receiver as there is also `LSP1UniversalReceiverDelegateVault` which i thought it was referring to